{{ cookiecutter.repo_name }}/app/src/main/java/{{ cookiecutter.package_dir }}/data/remote/apirequest/ApiResult.kt

```diff
@@ -0,0 +1,39 @@
+package {{ cookiecutter.package_name }}.data.remote.apirequest
+
+import {{ cookiecutter.package_name }}.data.remote.ApiService
+
+/**
+ * Interface to handle different results from a Retrofit API call.
```

Comment:
You can improve the wording to "Interface to wrap API results from a Retrofit API call"